update_copyright.d

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env rdmd
+
+/**
+Update the copyright notices in source files so that they have the form:
+---
+    Copyright XXXX-YYYY by The D Language Foundation, All Rights Reserved
+---
+It does not change copyright notices of authors that are known to have made
+changes under a proprietary license.
+
+Copyright:  Copyright (C) 2017-2018 by The D Language Foundation, All Rights Reserved
+
+License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
+
+Authors:    Iain Buclaw
+
+Example usage:
+---
+update-copyright.py --update-year src/dmd
```

Comment:
`.py` extension?

Reply:
Sometimes I forget which language I'm writing in. 😄